automation/patch_igz/patch_remote.py

```diff
@@ -386,90 +365,74 @@ def _wait_deployment_ready(self):
                 "app.kubernetes.io/name=mlrun",
                 "--for",
                 "condition=Ready",
-                "--timeout=240s",
+                "--timeout=300s",
             ],
             live=True,
         )
 
     def _reset_mlrun_db(self):
-        curr_worker_replicas = (
+        non_mlrun_db_services_deployment_selector = (
```

Comment:
now the `non_mlrun_db_services_deployment_selector` name  is incorrect 
Maybe - `non_mlrun_deployment_selector`?

Reply:
will use `mlrun_api_services_deployment_selector`